modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -71,9 +103,26 @@ public InfoFieldSet getInfoFieldSet(String itemClassName) {
 
 		List<InfoFieldValue<Object>> infoFieldValues = new ArrayList<>();
 
+		Set<AssetVocabulary> assetVocabularies = _getAssetVocabularies(
+			assetEntry);
+
+		Locale locale = LocaleThreadLocal.getThemeDisplayLocale();
+
+		for (AssetVocabulary assetVocabulary : assetVocabularies) {
+			infoFieldValues.add(
+				new InfoFieldValue<>(
+					new InfoField(
+						TextInfoFieldType.INSTANCE,
+						InfoLocalizedValue.localize(
+							getClass(), assetVocabulary.getTitle(locale)),
+						assetVocabulary.getName()),
+					() -> _getCategoryNames(
```

Comment:
Let's use the opportunity to think of a way to pass the actual AssetCategory objects as the value all the way to the frontend and decide there how to convert them to a string. That way we are opening the door to having a better representation than a comma separated list of the names. At the very least I hope that soon we'll be able to make the categories a link to the associated category page (once we have one).